Index: src/Users.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\r\nAlex Ewing|aewing24@uw.edu\r\n|\r\nlperry2|lperry2@uw.edu\r\nsv|vf \r\noo|oo@uw.edu\r\n\r\nsdflj|sldkfj\r\nsdfsd|sdfsdf\r\nsdf|sdf\r\nsdf|sdf\r\nsdflkj|sdfkj\r\nsdf|sdf\r\nsdf|sdf\r\nsdf|sdf\r\ndsf|sdfd\r\nsdf|sdfk\r\nsdf|sdfsd\r\nsdfs|sdfds\r\n|\r\n|\r\n|\r\n|\r\n|\r\ngjhbj,|\r\n|\r\nfre|fr3gr\r\n|\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Users.txt b/src/Users.txt
--- a/src/Users.txt	(revision b1882bfd0a551a41cb1424cc0f65a90a14340c02)
+++ b/src/Users.txt	(date 1716342130406)
@@ -26,3 +26,5 @@
 |
 fre|fr3gr
 |
+|
+|
Index: out/production/Group-Emerald/.idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"PDMPlugin\">\r\n    <option name=\"skipTestSources\" value=\"false\" />\r\n  </component>\r\n  <component name=\"ProjectRootManager\" version=\"2\" languageLevel=\"JDK_19\" default=\"true\" project-jdk-name=\"openjdk-22\" project-jdk-type=\"JavaSDK\">\r\n    <output url=\"file://$PROJECT_DIR$/out\" />\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/out/production/Group-Emerald/.idea/misc.xml b/out/production/Group-Emerald/.idea/misc.xml
--- a/out/production/Group-Emerald/.idea/misc.xml	(revision b1882bfd0a551a41cb1424cc0f65a90a14340c02)
+++ b/out/production/Group-Emerald/.idea/misc.xml	(date 1716341936623)
@@ -3,7 +3,7 @@
   <component name="PDMPlugin">
     <option name="skipTestSources" value="false" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_19" default="true" project-jdk-name="openjdk-22" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_22" default="true" project-jdk-name="openjdk-22" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/out" />
   </component>
 </project>
\ No newline at end of file
Index: out/production/Group-Emerald/src/Users.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\r\nAlex Ewing|aewing24@uw.edu\r\n|\r\ndfds|dbdsf\r\ncsdc|zvad\r\nzdfzvdfv|dfbdsf\r\nsrv|dvf\r\nzfvad|fvsadf\r\nfzd |dvfd\r\nvdfv|zvdf \r\ndf sd|xcf d\r\ngnb vg| bn\r\nsrva|rver \r\nsrv|dfv\r\nsrdv|dfb \r\nsfdrvs|df sd\r\ndfsdf|vadfs \r\ndfb|fb dg\r\ndzvf|fvzd \r\ndfb|bsds\r\nbfdxfb|xdfnd\r\nfdg fdg |fgdfg \r\ndfv|fdd s\r\nxdfv|xdbf \r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/out/production/Group-Emerald/src/Users.txt b/out/production/Group-Emerald/src/Users.txt
--- a/out/production/Group-Emerald/src/Users.txt	(revision b1882bfd0a551a41cb1424cc0f65a90a14340c02)
+++ b/out/production/Group-Emerald/src/Users.txt	(date 1716341969158)
@@ -1,24 +1,29 @@
 
 Alex Ewing|aewing24@uw.edu
 |
-dfds|dbdsf
-csdc|zvad
-zdfzvdfv|dfbdsf
-srv|dvf
-zfvad|fvsadf
-fzd |dvfd
-vdfv|zvdf 
-df sd|xcf d
-gnb vg| bn
-srva|rver 
-srv|dfv
-srdv|dfb 
-sfdrvs|df sd
-dfsdf|vadfs 
-dfb|fb dg
-dzvf|fvzd 
-dfb|bsds
-bfdxfb|xdfnd
-fdg fdg |fgdfg 
-dfv|fdd s
-xdfv|xdbf 
+lperry2|lperry2@uw.edu
+sv|vf 
+oo|oo@uw.edu
+
+sdflj|sldkfj
+sdfsd|sdfsdf
+sdf|sdf
+sdf|sdf
+sdflkj|sdfkj
+sdf|sdf
+sdf|sdf
+sdf|sdf
+dsf|sdfd
+sdf|sdfk
+sdf|sdfsd
+sdfs|sdfds
+|
+|
+|
+|
+|
+gjhbj,|
+|
+fre|fr3gr
+|
+|
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"PDMPlugin\">\r\n    <option name=\"skipTestSources\" value=\"false\" />\r\n  </component>\r\n  <component name=\"ProjectRootManager\" version=\"2\" languageLevel=\"JDK_19\" default=\"true\" project-jdk-name=\"openjdk-22\" project-jdk-type=\"JavaSDK\">\r\n    <output url=\"file://$PROJECT_DIR$/out\" />\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	(revision b1882bfd0a551a41cb1424cc0f65a90a14340c02)
+++ b/.idea/misc.xml	(date 1716341936623)
@@ -3,7 +3,7 @@
   <component name="PDMPlugin">
     <option name="skipTestSources" value="false" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_19" default="true" project-jdk-name="openjdk-22" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_22" default="true" project-jdk-name="openjdk-22" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/out" />
   </component>
 </project>
\ No newline at end of file
