Index: src/Users.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\r\nAlex Ewing|aewing24@uw.edu\r\n|\r\nlperry2|lperry2@uw.edu\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Users.txt b/src/Users.txt
--- a/src/Users.txt	(revision 0d8566cf651c5521e7f05f9d3bcd7a6f6b8b2f7a)
+++ b/src/Users.txt	(date 1715911580574)
@@ -2,3 +2,4 @@
 Alex Ewing|aewing24@uw.edu
 |
 lperry2|lperry2@uw.edu
+|
Index: out/production/Group-Emerald/src/Users.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\r\nAlex Ewing|aewing24@uw.edu\r\n|\r\ndfds|dbdsf\r\ncsdc|zvad\r\nzdfzvdfv|dfbdsf\r\nsrv|dvf\r\nzfvad|fvsadf\r\nfzd |dvfd\r\nvdfv|zvdf \r\ndf sd|xcf d\r\ngnb vg| bn\r\nsrva|rver \r\nsrv|dfv\r\nsrdv|dfb \r\nsfdrvs|df sd\r\ndfsdf|vadfs \r\ndfb|fb dg\r\ndzvf|fvzd \r\ndfb|bsds\r\nbfdxfb|xdfnd\r\nfdg fdg |fgdfg \r\ndfv|fdd s\r\nxdfv|xdbf \r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/out/production/Group-Emerald/src/Users.txt b/out/production/Group-Emerald/src/Users.txt
--- a/out/production/Group-Emerald/src/Users.txt	(revision 0d8566cf651c5521e7f05f9d3bcd7a6f6b8b2f7a)
+++ b/out/production/Group-Emerald/src/Users.txt	(date 1715911547497)
@@ -1,24 +1,4 @@
 
 Alex Ewing|aewing24@uw.edu
 |
-dfds|dbdsf
-csdc|zvad
-zdfzvdfv|dfbdsf
-srv|dvf
-zfvad|fvsadf
-fzd |dvfd
-vdfv|zvdf 
-df sd|xcf d
-gnb vg| bn
-srva|rver 
-srv|dfv
-srdv|dfb 
-sfdrvs|df sd
-dfsdf|vadfs 
-dfb|fb dg
-dzvf|fvzd 
-dfb|bsds
-bfdxfb|xdfnd
-fdg fdg |fgdfg 
-dfv|fdd s
-xdfv|xdbf 
+lperry2|lperry2@uw.edu
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"PDMPlugin\">\r\n    <option name=\"skipTestSources\" value=\"false\" />\r\n  </component>\r\n  <component name=\"ProjectRootManager\" version=\"2\" languageLevel=\"JDK_20\" default=\"true\" project-jdk-name=\"openjdk-22\" project-jdk-type=\"JavaSDK\">\r\n    <output url=\"file://$PROJECT_DIR$/out\" />\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	(revision 0d8566cf651c5521e7f05f9d3bcd7a6f6b8b2f7a)
+++ b/.idea/misc.xml	(date 1716339276934)
@@ -3,7 +3,7 @@
   <component name="PDMPlugin">
     <option name="skipTestSources" value="false" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_20" default="true" project-jdk-name="openjdk-22" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_22" default="true" project-jdk-name="openjdk-22" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/out" />
   </component>
 </project>
\ No newline at end of file
